sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -623,11 +623,11 @@ static DocumentSentiment getExpectedDocumentSentiment() {
         SentenceOpinionPropertiesHelper.setAssessments(sentenceOpinion2,
             new IterableStream<>(asList(assessmentSentiment3)));
         final SentenceSentiment sentenceSentiment1 = new SentenceSentiment(
-            "The hotel was dark and unclean.", TextSentiment.NEGATIVE,
+            "The hotel was dark and unclean. ", TextSentiment.NEGATIVE,
```

Comment:
This revision result is not an expected result. The reported issue is still in open status. But thinking "suffix with empty space" is not a big concern so I leave it as-is, accept the current service behavior, and expect to fail this test after the service fixes it.

Reply:
https://dev.azure.com/msazure/Cognitive%20Services/_workitems/edit/14208842

with accepting what it has, I have more test coverage since most the sentiment analysis depends on the input.

Reply:
Thanks for the explanation, will mark this as resolved.